.travis/stages/verify/check-links-and-yaml-syntax/check-links

```diff
@@ -15,28 +17,31 @@ fi
 
 function checkUri() {
   local -r uri="$1"
-  
-  local -r curlResult=$(curl -s --head "$uri" | head -n 1)
-  echo "$uri -- $curlResult"  
-  # Temporarily disabled. We're unable to connect to A&A.org from Travis.
-
-  # echo $curlResult |  grep -qE  "HTTP/2 200|(200|302) (OK|Found)" \
-  #  || reportBrokenUri "$uri"
+  local output
+  if ! output="$FAILURE_PREFIX$(curl -Lsf -o /dev/null -w "%{http_code}" "$uri")"; then
+    reportBrokenUri "$uri" "$output"
+    FAILURE=1
+  fi
 }
 
 function reportBrokenUri() {
-  FAILURE=1 
   local -r red="\e[38;5;196m"
   local -r blue="\e[38;5;75m"
-  echo -en "Error: ${red}$uri${end} ${blue}"
-  curl -s --head "$uri" | head -n 1 | sed 's|HTTP/[12][.01]?||'
-  echo -en "${end}"
+  local -r error_reason=${2//${FAILURE_PREFIX}000/Connection Failure!}
```

Comment:
This replacement expression replaces `Failed With Code: 000` with `Connection Failure`, which would be printed otherwise if curl was unable to reach the server at all

Reply:
I'd recommend, if possible, replacing the '000' with 'Connection Failure' and then prefixing the 'Failed with Code' otherwise. Matching this much of a string seems indirect to me. I also think some commentary in-script would go quite a long way to explain that.

Reply:
I thought this way round is pretty convenient from a code perspective because it avoids the need for an else clause, but I think I can write something self-explanatory there